zerver/lib/markdown/__init__.py

```diff
@@ -2149,6 +2148,11 @@ def build_inlinepatterns(self) -> markdown.util.Registry:
         # space for us to add our own.
         reg = markdown.util.Registry()
         reg.register(BacktickInlineProcessor(markdown.inlinepatterns.BACKTICK_RE), "backtick", 105)
+        reg.register(
+            markdown.inlinepatterns.EscapeInlineProcessor(markdown.inlinepatterns.ESCAPE_RE, self),
+            "escape",
+            102,
+        )
```

Comment:
How did you pick 102?  Might be worth commenting on how that's done in python-markdown upstream.